src/mono/wasm/testassets/WasmBasicTestApp/App/MiscTests.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.JavaScript;
+
+public partial class MiscTests
+{
+    [JSExport]
+    public static unsafe int TestGetFunctionPointerForDelegate()
```

Comment:
why does this need to be a WABT? couldn't we just add it to https://github.com/dotnet/runtime/blob/f9c5e1a89dda21e672a859d13197897b1bb1cd4f/src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/GetFunctionPointerForDelegateTests.cs as a PlatformSpecific test?

most of the tests there are disabled against https://github.com/dotnet/runtime/issues/39187 so we should close that issue too and change the tests from ActiveIssue to `[SkipOnPlatform(TestPlatforms.Browser | TestPlatforms.Wasi, "Not supported on Browser/WASI.")]`, though looking at the tests I wonder if we run them on Wasi since they're not skipped there from what I can see.